dev-docs/infra-server-iam.md

```diff
@@ -42,4 +42,9 @@ Specify the action to restrict user access to the specific action.
 | Get Infra Server | GET | infra:infraServers:get | /api/v0/infra/servers/{id} | https://{{< example_fqdn "automate" >}}/api/v0/infra/servers/{id} |
 | Create Infra Server | POST | infra:infraServers:create | /api/v0/infra/servers | https://{{< example_fqdn "automate" >}}/api/v0/infra/servers |
 | Update Infra Server | PUT | infra:infraServers:update | /api/v0/infra/servers/{id} | https://{{< example_fqdn "automate" >}}/api/v0/infra/servers/{id} |
-| Delete Infra Server | DELETE | infra:infraServers:delete | /api/v0/infra/servers/{id} | https://{{< example_fqdn "automate" >}}/api/v0/infra/servers/{id} |
\ No newline at end of file
+| Delete Infra Server | DELETE | infra:infraServers:delete | /api/v0/infra/servers/{id} | https://{{< example_fqdn "automate" >}}/api/v0/infra/servers/{id} |
+| List Orgs | GET | infra:infraServersOrgs:list | /api/v0/infra/servers/{server_id}/orgs | https://{{< example_fqdn "automate" >}}/api/v0/infra/servers/{server_id}/orgs |
+| Get Org | GET | infra:infraServersOrgs:get | /api/v0/infra/servers/{server_id}/orgs/{id} | https://{{< example_fqdn "automate" >}}/api/v0/infra/servers/{server_id}/orgs/{id} |
+| Create Org | POST | infra:infraServersOrgs:create,iam:projects:assign | /api/v0/infra/servers/{server_id}/orgs | https://{{< example_fqdn "automate" >}}/api/v0/infra/servers/{server_id}/orgs |
```

Comment:
Why do we have the projects:assign?

Reply:
list and get are different actions, if we want to display only list of orgs but not details inside any particular org, in this case we can make use of infra:infraServersOrgs:list. But if we want user to see the list of orgs as well as details inside each org then we need to provide infra:infraServersOrgs:list and infra:infraServersOrgs:get both the actions to user only infra:infraServersOrgs:get will not make sense in UI. But if retrieval of org is via api then we can have assign only infra:infraServersOrgs:get to api-token.

Regarding projects:assign? 
As we are assigning project too while creating org, thus any user with infra:infraServersOrgs:create action needs to have iam:projects:assign action without it user cannot create org, as iam:projects:assign action is a dependency to create orgs.